slack.json

```
@@ -1,5 +1,6 @@
 {
   "hooks": {
-    "get-hooks": "npx -q --no-install -p @slack/cli-hooks slack-cli-get-hooks"
+    "get-hooks": "npx -q --no-install -p @slack/cli-hooks slack-cli-get-hooks",
+    "deploy": "echo 'todo: replace this in slack.json!' && exit 1"
```

```suggestion
    "deploy": "node -e 'throw new Error(`TODO: replace this in slack.json`)'"
```

Here's another option that might be more stable across platforms 👀